apps/frontend/src/app/Data/Characters/RaidenShogun/index.tsx

```diff
@@ -105,12 +105,16 @@ const skillEye_ = equal(
     })
   )
 )
-
+const [condInBurstPath, condInBurst] = cond(key, 'InBurst')
 function skillDmg(atkType: number[]) {
   // if Raiden is above or equal to C2, then account for DEF Ignore else not
   return dmgNode('atk', atkType, 'skill', {
     enemy: {
-      defIgn: greaterEq(input.constellation, 2, dm.constellation2.def_ignore),
+      defIgn: greaterEq(
```

Comment:
This should be in the general premod so that all dmg instances are affected by this

Reply:
I tried doing this. It's in my stashed changes; Idk if I did something wrong, but the damages didn't match when I tried running it. Gonna commit so you can tell me if I did anything wrong 😳